e2e_deployment_files/nestededge_middleLayerBaseDeployment_amqp.json

```diff
@@ -63,30 +63,6 @@
                 "value": "443"
               }
             }
-          },
-          "relayer1": {
-            "version": "1.0",
-            "type": "docker",
-            "status": "running",
-            "restartPolicy": "always",
-            "env": {
-              "transportType": {
-                "value": "Amqp"
-              },
-              "inputName": {
-                "value": "input1"
-              },
-              "outputName": {
-                "value": "output1"
-              },
-              "EnableTrcReporting": {
-                "value": false
-              }
-            },
-            "settings": {
-              "image": "$upstream:443/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
-              "createOptions": ""
-            }
```

Comment:
Taking this test module out of the middle level deployment, as this will now serve as the basic deployment with no test modules. I have added a whole separate deployment for these more complex deployments.